microservices/recommendation/src/main/resources/application.yml

```
@@ -0,0 +1,101 @@
+---
+spring.config.activate.on-profile: streaming_instance_1
+
+spring.cloud.stream.bindings.messageProcessor-in-0.consumer.instanceIndex: 1
+---
+
+spring.config.activate.on-profile: kafka
+
+management.health.rabbit.enabled: false
+spring.cloud.stream.defaultBinder: kafka
+spring.kafka.bootstrap-servers: kafka:9092
+spring.cloud.stream.kafka.binder.replication-factor: 1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)